kalite/testing/mixins/store_mixins.py

```diff
@@ -7,8 +7,8 @@ class CreateStoreTransactionLogMixin(object):
         'item': StoreItem.all().values()[0].storeitem_id,
         'value': -1*StoreItem.all().values()[0].cost,
         'purchased_at': datetime.datetime.now(),
-        'context_type': 'unit',
```

Comment:
Can probably delete this file as well, as we should delete the store app and hence not be testing it.